src/js/post-911-gib-status/components/EnrollmentHistory.jsx

```diff
@@ -22,28 +21,38 @@ class EnrollmentHistory extends React.Component {
     */
 
     // History explanation box
+    let historyExplanationBox;
     const benefitMaybePending =
-      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement > 0) &&
-      enrollmentData.usedEntitlement === 0;
+      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement.months > 0) &&
```

Comment:
Yeah, OR sounds right. But I'm confused about why `benefitMaybePending` disappeared. Will pore over it some more to understand.